.github/workflows/docker.yml

```diff
@@ -59,4 +60,23 @@ jobs:
           # Docker compose can take some time to start
           max_attempts: 5
           timeout_minutes: 2
-          command: sleep 60 && linera --wallet docker/wallet.json --storage rocksdb:docker/linera.db sync-balance
\ No newline at end of file
+          command: sleep 60 && linera --wallet docker/wallet.json --storage rocksdb:docker/linera.db sync-balance
+      - name: Run benchmarks
+        run: |
+          ./scripts/bench.sh docker/wallet.json rocksdb:docker/linera.db > benchmark_results.txt
+      - name: Read benchmark results
+        id: read_results
+        run: |
+          results=$(cat benchmark_results.txt)
+          echo "::set-output name=results::$results"
+      - name: Post benchmark results as a PR comment
+        uses: peter-evans/create-or-update-comment@v3
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          repository: ${{ github.repository }}
+          issue-number: ${{ github.event.pull_request.number }}
+          body: |
+            ## Benchmark Results
+            ```
+            ${{ steps.read_results.outputs.results }}
+            ```
```

Comment:
Newlines at ends of files please!  `.editorconfig` should make this happen by default if your editor is reading it properly.  Ditto the other file.